### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by lidiyag**
This PR updates the README file for katacoda-scenarios.

### CODE REVIEW

**Comment by lidiyag** [comment]
cc @sids-b @fisherxu

**Comment by fisherxu** [comment]
lgtm  thanks! @lidiyag

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
 # KubeEdge Katacoda-Scenarios
+
+We have created a tutorial in the interactive learning platform Katacoda for kubeedge deployment. This can give a hands-on experience of kubeedge deployment. The tutorial is created on kubeedge relese v1.0.0.
```
Comment (rohitsardesai83) [inline README.md]: kubeedge -> KubeEdge

Comment (rohitsardesai83) [reply README.md]: relese -> release

Comment (lidiyag) [reply README.md]: Updated

File: README.md [hunk 2]
```diff
@@ -1 +1,10 @@
 # KubeEdge Katacoda-Scenarios
+
+We have created a tutorial in the interactive learning platform Katacoda for KubeEdge deployment. This can give a hands-on experience of kubeedge deployment. The tutorial is created on kubeedge release v1.0.0.
```
Comment (rohitsardesai83) [inline README.md]: created on *kubeedge* release v1.0.0-> created on *KubeEdge* release v1.0.0

Comment (lidiyag) [reply README.md]: Updated

**Review by sids-b — APPROVED** [review]

**Pull request merged**